contracts/database/orm/orm.go

```diff
@@ -92,6 +92,8 @@ type Query interface {
 	Omit(columns ...string) Query
 	// Order specifies the order in which the results should be returned.
 	Order(value any) Query
+	// OrderBy specifies the order should be ascending.
+	OrderBy(column string) Query
```

Comment:
S: Add `direction` parameter, and default `asc`.

```suggestion
	OrderBy(column string, direction ...string) Query
```